INSTALL.md

```diff
@@ -52,5 +52,24 @@ On Linux, you will probably need to install `ncurses` manually. On Ubuntu, for
 example, this can be done by running:
 
 ```
-$ sudo apt-get install libncurses5-dev
+$ sudo apt install libncurses5-dev
 ```
+
+Another `curses` dependency may be `libtinfo`. If you encounter this error:
+```
```

Comment:
This makes it sound like it's a different problem from the one listed above, but it's not; `libncurses5-dev` depends on `libtinfo5`, so installing `libncurses5-dev` is enough to fix the issue. Adding the "error while loading shared libraries" message here would be good so that it's easier for people to see whether this is the problem they're running into, but other than that I think this section should be left alone.

Reply:
As demonstrated in https://github.com/purescript/npm-installer/pull/26#issuecomment-657253719 `libtinfo5` is required. Installing `libncurses5-dev` will not fix this error.

Reply:
This is still the same issue; what's happening here is that libncurses5-dev depends on libtinfo6 in Ubuntu 20.04, whereas it depended on libtinfo5 in previous Ubuntu versions; see https://packages.ubuntu.com/focal/libtinfo6. I still don't want to list  two separate fixes here. Since `libtinfo5` is the package which provides the actual file we need (specifically `/lib/x86_64_linux_gnu/libtinfo.so.5`) hopefully we can just instruct users to install that. I'm testing to see whether instructing users to install `libtinfo5` will work on all supported Ubuntu versions.

Reply:
I've also just remembered that whatever we specify here needs to work for building the compiler from source too. From my testing, it seems `libncurses5-dev` is needed for compiling the compiler from source (in which case you end up with a `purs` which depends on libtinfo6), but `libtinfo5` is needed for using the prebuilt binaries, since those depend on libtinfo5. I think we could just ask the user to install both:
```
$ sudo apt install libtinfo5 libncurses5-dev
```

Reply:
Interesting. Nice follow-up investigation.

> I still don't want to list two separate fixes here. I think we could just ask the user to install both.

These seem like two separate dependency problems though (stack build versus npm download). I think it's better to only ask users to install the dependencies they actually need.

Reply:
Building from source isn't separate from the npm installer though: the npm installer falls back to building from source if the prebuilt binaries fail. Systems where the prebuilt binaries will never work aren't that uncommon; they won't work on any architecture other than x86_64, and they link against glibc which means that they don't work on any Linux distribution not based on glibc, such as Alpine. Additionally, the `libncurses5-dev` package is only about 1MB when installed.

Reply:
Incidentally we can also avoid this entirely by dropping the `libtinfo` dependency. See https://github.com/purescript/purescript/issues/3696#issuecomment-657282303